networkx/algorithms/bipartite/tests/test_matching.py

```diff
@@ -82,3 +84,13 @@ def test_to_vertex_cover(self):
         matching = maximum_matching(self.graph)
         vertex_cover = to_vertex_cover(self.graph, matching)
         self.check_vertex_cover(vertex_cover)
+
+
+def test_eppstein_matching():
+    # Special test
```

Comment:
I also sometimes have that issue with nose in certain environments, and I don't know why it does that.

How about just `#` comments above for those that require some explanation or a reference to a GitHub issue number?